src/main/resources/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction/tableQueue/table.properties

```
@@ -0,0 +1,46 @@
+# The MIT License
+#
+# Copyright 2023 Martin Pokorny.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+queue.isEmpty=The queue is currently empty.
+queue.warning.count=The queue has {0} item(s). The oldest one starts {1} ago!
+queue.warning.count.detail=\
+In many cases, users have forgot to un-reserve the resource (or an external API has died). \
+Sometimes the build does not release the resource. \
+This can happen when Jenkins crashes and the builds are no longer executable after restart.<br>\
+Please check the status and manually release the resource if necessary.
+If you do not have permission to release the resource, contact your administrator.
```

I don't like "starts" very much because the request is actually in queue, so in my opinion it is rather "waiting" than "started".
```suggestion
queue.warning.count=The queue has {0} item(s). The oldest one was inserted {1} ago!
queue.warning.count.detail=\
In many cases, users forget to un-reserve the resource (or an external API has died). \
Sometimes the build does not release the resource. \
This can happen when Jenkins crashes and the builds are no longer executable after restart.<br>\
Please check the status and manually release the resource if necessary.
If you do not have permission to release the resource, contact your administrator.
```